src/utils/common/properties.ts

```diff
@@ -0,0 +1,3 @@
+export function isEmpty(value: string | undefined): boolean {
+  return typeof value !== 'undefined' && !value.trim().length;
+}
```

Comment:
isEmpty only checks if a string is defined right? if so maybe we should name it something more specific